.github/workflows/ci-core.yml

```diff
@@ -31,14 +31,16 @@ jobs:
       run: helm lint keda
 
   deploy-helm-3-x:
-    name: Deploy to Kubernetes ${{ matrix.kubernetesVersion }} (${{ (matrix.enableAzureWorkloadIdentity == true && 'With Azure Workload Identity') || 'Without Azure Workload Identity' }})
+    name: Deploy to Kubernetes ${{ matrix.kubernetesVersion }} (${{ (matrix.enableAzureWorkloadIdentity == true && 'With Azure Workload Identity') || 'Without Azure Workload Identity' }}) (${{ (matrix.enableCertManager == true && 'With cert-manager') || 'Without cert-manager' }}) in namespace '${{matrix.namespace}}'
```

Comment:
Nit - We should only have 1 `()` as it looks funky atm.

Reply:
Could you open a PR changing it?